src/integration/lightning/lightning-client.ts

```diff
@@ -216,6 +222,16 @@ export class LightningClient {
     );
   }
 
+  async updateLnurlpLink(linkId: string, data: LnurlpLinkUpdateDto): Promise<LnurlpLinkDto> {
+    if (!linkId) throw new Error('LinkId is undefined');
+
+    return this.http.post<LnurlpLinkDto>(
+      `${Config.blockchain.lightning.lnbits.lnurlpApiUrl}/links?link_id=${linkId}`,
+      data,
+      this.httpLnBitsConfig(),
+    );

```

Comment:
Why no use the update endpoint (PUT)?
https://vm-dfx-btc-inp-loc.westeurope.cloudapp.azure.com:5000/docs#/lnurlp/api_link_create_or_update_lnurlp_api_v1_links__link_id__put

Reply:
Just saw the first entry "Api Link Create Or Update" and used it :-)
![grafik](https://github.com/DFXswiss/api/assets/104787072/6df52f4c-b1a7-42ba-9e2f-c63cf4583383)

Reply:
But you are right - the "put" endpoint makes more sense for update purposes